test/test.events-no-postroll.js

```diff
@@ -32,9 +32,10 @@ QUnit.module('Final Events With No Postroll', {
 
 QUnit.test('final ended event with no postroll: just 1', function(assert) {
   var done = assert.async();
-
   var endedEvents = 0;
 
+  this.player.ads.settings.postrollTimeout = 1;
```

Comment:
why was this necessary?

Reply:
The test times out otherwise. This is just so the test runs faster basically. I'll add a comment.